modules/xcm-bridge-hub-router/src/lib.rs

```diff
@@ -100,67 +103,111 @@ pub mod pallet {
 	#[pallet::hooks]
 	impl<T: Config<I>, I: 'static> Hooks<BlockNumberFor<T>> for Pallet<T, I> {
 		fn on_initialize(_n: BlockNumberFor<T>) -> Weight {
-			// if XCM queue is still congested, we don't change anything
-			if T::WithBridgeHubChannel::is_congested() {
-				return T::WeightInfo::on_initialize_when_congested()
-			}
+			Bridge::<T, I>::mutate(|bridge| {
+				// TODO: make sure that `WithBridgeHubChannel::is_congested` returns true if either
```

Comment:
Relevant commit: https://github.com/paritytech/cumulus/commit/c0e3225cca573a7ee64180e3bedade6048a9c945. I'll remove this TODO before merging